tests/test_multiprocessing.py

```diff
@@ -1,9 +1,11 @@
+from __future__ import print_function
+
 import os
 import math
 import time
 import pickle
 import pprint
-import random
+from random import random, choice
```

Comment:
Any strong opinion here? I agree `random.random()` is ugly, but I don't like the stand-alone `choice()`. Can we keep the `import random`? I generally don't like `from ... import ...`.